### PR TITLE
Simplify the configuration of everserver

### PR DESCRIPTION
**Pull request by oyvindeide**
Everserver is installed as a console script, which means we dont need everserver.py to be executable. This makes it more transparent which entry point we are actually using.

### CODE REVIEW

File: src/everest/strings.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 DEFAULT_OUTPUT_DIR = "everest_output"
 DEFAULT_LOGGING_FORMAT = "%(asctime)s %(name)s %(levelname)s: %(message)s"
 
-EVEREST_SERVER = "everserver"
+EVEREST_SERVER = "fm_everserver"
```
Comment (verveerpj) [inline src/everest/strings.py:8]: Should that also be renamed in the script entry point in `pyproject.toml`?

**Review by verveerpj — APPROVED** [review]
LGTM, one comment.

File: src/everest/strings.py (hunk 1 above)
Comment (oyvindeide) [reply src/everest/strings.py:8]: Maybe, maybe it would be better to rename the script file instead? Might make it less confusing.

Comment (oyvindeide) [reply src/everest/strings.py]: As it sits now we are really adding a config here, as that is the ert way, though I would like to refactor this to just submit directly, and not go through the ert config which is a bit cumbersome and not very easy to read.

**Pull request merged**